apps/settings/style/screen_lock.css

```diff
@@ -168,12 +168,12 @@ span.cb-passcode-digit[data-dot]::before {
   content: '';
   display: block;
   position: absolute;
-  width: 1.5rem;
-  height: 1.5rem;
+  width: 0.7rem;
+  height: 0.7rem;
   background-color: #3e3b39;
   border-radius: 0.75rem;
-  top: 50%;
-  left: 50%;
+  top: 64%;
```

Comment:
It's should be in % only, because when we use rem then When we change the screen size it's not come in center, for responsive design it is in % only.